packages/client/hmi-client/src/services/knowledge.ts

```diff
@@ -179,4 +179,23 @@ export const extractPDF = async (artifact: Artifact) => {
 	}
 };
 
+export async function codeToAMR(codeId: string) {
+	const response = await API.post(`/extract/code-to-amr?code_id=${codeId}`);
```

Comment:
if `name` and `description` are optional, why not remove them from our backend?
Should we let the user set them up when clicking on the call?
```suggestion
	const response = await API.post(`/knowledge/code-to-amr?code_id=${codeId}`);
```